app/services/allocation_service.rb

```diff
@@ -46,16 +52,30 @@ def self.active_allocations(nomis_offender_ids)
   end
 
   def self.previously_allocated_poms(nomis_offender_id)
-    Allocation.inactive_allocations(nomis_offender_id).
-      map(&:primary_pom_nomis_id)
+    allocation = AllocationVersion.find_by(nomis_offender_id: nomis_offender_id)
+
+    if allocation.nil?
```

Comment:
If this simpler as 

```
return [] if allocation.nil?
```

as it'll remove the need for the if/else

Reply:
👍